components/esp_hw_support/include/esp_ds.h

```diff
@@ -86,10 +86,10 @@ typedef struct esp_digital_signature_data {
  * The encryption is a prerequisite step before any signature operation can be done.
  */
 typedef struct {
-    uint32_t  Y[ESP_DS_SIGNATURE_MAX_BIT_LEN / 32]; //!< RSA exponent
-    uint32_t  M[ESP_DS_SIGNATURE_MAX_BIT_LEN / 32]; //!< RSA modulus
-    uint32_t Rb[ESP_DS_SIGNATURE_MAX_BIT_LEN / 32]; //!< RSA r inverse operand
-    uint32_t M_prime;                               //!< RSA M prime operand
+    uint32_t  Y[ESP_DS_SIGNATURE_MAX_BIT_LEN / 32]; //!< RSA exponent, little endian
```

Comment:
I dont think it is necessary to explicitly mention the format here, but I think it can surely be helpful additional to the note section.